tests/io/dask/delayed/test_delete.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 from kartothek.io.dask.delayed import delete_dataset__delayed
+from kartothek.io.eager import garbage_collect_dataset
```

Comment:
For the dask tests we should also use the dask function

Reply:
you mean dask garbage collection function ?

Reply:
yes